site/content/en/docs/Reference/Networking/proxy.md

```diff
@@ -40,7 +40,7 @@ To make the exported variables permanent, consider adding the declarations to ~/
 ```shell
 set HTTP_PROXY=http://<proxy hostname:port>
 set HTTPS_PROXY=https://<proxy hostname:port>
-set NO_PROXY=localhost,127.0.0.1,10.96.0.0/12,192.168.99.1/24,192.168.39.0/24
```

Comment:
shouldn't that be irrelevant the 0 or 1 because it is /24 ?

Reply:
Yes I think that is the case, but I thought it would be nicer to align it with the other examples.